tests/regression/common/file-upload.test.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from '@playwright/test';
+import * as dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config();
+
+test('file upload test', async ({ page }) => {
+  await page.goto('https://monkkee.com/app#/');
```

Comment:
According to the task description you should use Page Object pattern in your tests - please do this.

Reply:
It means that you tests should consist of PO methods (actions) and assertions (expect) like:
```
....
loginPage.open();
loginPage.login(email, pass);
expect(homePage.logo).toBeVisible()
....
```